README.md

```diff
@@ -18,3 +18,19 @@ See the help for the `bioconda-utils` command-line interface for details:
 ```
 bioconda-utils -h
 ```
+
+## Changelog
+
+- recipes can be built in a docker container or by the system conda-build using the same infrastructure
+- improved log files for easy grepping (e.g., `grep BIOCONDA log`)
+- given a subdag of recipes to build, if one recipe fails then other recipes
+  that depend on it will be skipped (and noted in the log).
+- uses the API from conda 2.0+ and in general works with the latest conda-build
+  versions (instead of maintaining our own branch)
+- test suite for the build tools
+- documentation
+- each built package can now be tested in an isolated busybox container thanks
+  to `mulled-build` and `involucro`. This will catch issues where a recipe fails
+  to specify libs (e.g., libgcc) in the run dependencies.
+- the `mulled-build` testing also means we get per-package docker containers for free
```

Comment:
We need to implement `push` at some point to get the containers into biocontainers. I will do this after this PR is merged and working upstream.